lib/get-config.js

```diff
@@ -283,6 +286,51 @@ function processIgnores(config) {
   config.ignore = config.ignore.map((pattern) => micromatch.matcher(pattern));
 }
 
+/**
+ * we were passed a rule, add the rule being passed in, to the config.
+ * ex:
+ * rule:severity
+ * no-implicit-this:["error", { "allow": ["some-helper"] }]
+ */
+function getRuleFromString(rule) {
+  const indexOfSeparator = rule.indexOf(':') + 1;
+
+  // we have to split based on the index of the first instance of the separator because the separator could exist in the second half of the rule
+  const name = rule.substring(0, indexOfSeparator - 1);
+  let severity = rule.substring(indexOfSeparator);
+  let config = true;
+
+  // if we get passed an object that looks like no-implicit-this:["error", { "allow": ["some-helper"] }] we should split it accordingly
+  try {
+    const options = JSON.parse(severity);
+
+    severity = options[0];
+    config = options[1];
+  } catch (ex) {
+    // noop
+  }
+
+  let _severity = 0;
+
+  switch (severity) {
+    case 'error':
+      _severity = ERROR_SEVERITY;
+      break;
+    case 'warn':
+      _severity = WARNING_SEVERITY;
+      break;
+    case 'off':
+    default:
+      _severity = IGNORE_SEVERITY;
+  }
```

Comment:
I think we already have a method that does this. All we should need to do is pass the config to`determineRuleConfig` right?

https://github.com/ember-template-lint/ember-template-lint/blob/0a336f262008346fd55e49f27f4ffa5d79587982/lib/get-config.js#L401-L423

Reply:
I didn't see this! Resolved this!